serverless/verify-backup/scripts/verify-backup.sh

```diff
@@ -53,4 +53,4 @@ RESTORE_LOG="Successfully restored $DUMP_VERSION db dump! \
 notify "$RESTORE_LOG"
 
 # Completed Cronitor job
-curl "https://cronitor.link/$CRONITOR_CODE/complete" -m 10 || true
\ No newline at end of file
+curl "https://cronitor.link/$CRONITOR_CODE/complete" -m 10 || true
```

Comment:
Maybe we should move the corresponding Cronitor `/run` call into this script also? Otherwise, consecutive POST to the server endpoint will have a complete event without a corresponding start event on Cronitor.

Although, with that said, this mostly only happen in development since the container should also be started when the request from the scheduler comes in.

Reply:
Yes I think it makes more sense to start the cronitor job within the verify-backup script, else if there is a lag between the server start and the verify-backup script start, we might get false job failures.